tests/PyzTest/Zed/Touch/_support/_generated/TouchBusinessTesterActions.php

```diff
@@ -1,4 +1,4 @@
-<?php  //[STAMP] 99e6a882832f8b945d89c5c30e28bc59
+<?php  //[STAMP] a7fcba56a03d5632223c74c955125c14
```

Comment:
@stereomon Can we double check somehow which folders have been missing the gitignore for _generated folders?